modules/eco-poison-pill-operator-about.adoc

```diff
@@ -44,13 +44,13 @@ spec:
 +
 If a watchdog device is unavailable, the `PoisonPillConfig` CR uses a software reboot. 
 <3> Specify if you want to enable software reboot of the unhealthy nodes. By default, the value of `isSoftwareRebootEnabled` is set to `true`. To disable the software reboot, set the parameter value to `false`.
-<4> Specify the timeout duration to check connectivity with each API server. When this duration elapses, the Operator starts remediation.
-<5> Specify the frequency to check connectivity with each API server.
+<4> Specify the timeout duration to check connectivity with each API server. When this duration elapses, the Operator starts remediation. The timeout duration must be more than or equal to 10ms.
```

Comment:
I'm finding "timeout duration" confusing. Is it the time until timeout occurs (most likely) or the time a timeout lasts? Maybe this can be reworded?

Also, if the values next to the spec items in the screen capture are the timeout duration, that should be called out. (If these are the timeout durations, the values don't seem aligned with the minimums stated in the following.)

Reply:
Hi @nalhadef "timeout duration" (or duration of the timeout) is the amount of time that a timeout lasts

> Also, if the values next to the spec items in the screen capture are the timeout duration, that should be called out. (If these are the timeout durations, the values don't seem aligned with the minimums stated in the following.)
As discussed, these are sample values and don't need to be aligned